packages/backend/src/tests/propertyBased/properties/cardinalityMany.test.ts

```diff
@@ -33,6 +33,12 @@ describe('cardinalityMany', async () => {
             expect(Array.isArray(result)).toEqual(true);
 
             expect(result.length).toBeLessThanOrEqual(Number(query.limit));
+
+            for (const item of result) {
+                for (const column of Object.keys(query.select)) {
```

Comment:
This is not exactly true.

In theory you could have a query with `select: {id:false}` so just the prescence of the key is not enough.

A more precise test should be

```ts
const expectedKeys = Object.entries(query.select).flatMap(([key,selected]) => {
  return selected ? [key] : []
});

for (const item of result) {
  const actualKeys = Object.keys(item);
  expect(actualKeys).to.containSubset(expectedKeys)
}
```

This will also give a nicer error message if something breaks.